persimmon/website/templates/pages/login.html

```diff
@@ -0,0 +1,5 @@
+{% extends "main.html" %}
+{% block body %}
+    {% include "magic_form.html" with form=form api=api success=success %}
+    <script src="../../static/keyboard.js"></script>
```

Comment:
Can you use `{% static "keyboard.js" %}` so we don't have to hardcode path relationships?

Reply:
will change that